app/controllers/referee_interface/reference_controller.rb

```diff
@@ -33,10 +33,16 @@ def submit_feedback
       end
     end
 
-    def finish; end
-
     def confirmation; end
 
+    def confirm_consent
+      consent_to_be_contacted = params[:application_reference][:consent_to_be_contacted]
```

Comment:
I think this will blow up if the user doesn't choose anything and therefore `params[:application_reference]` is missing.